File: BEE 1013.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(' ');

var a = parseInt(lines.shift());
var b = parseInt(lines.shift());
var c = parseInt(lines.shift());

var maiorAB = (a+b+Math.abs(a-b))/2;
var greatestValue = (maiorAB+c+Math.abs(maiorAB-c))/2;

console.log(greatestValue+" eh o maior");

File: BEE 1013.py
import  math
a, b, c = map(int, input().split())
maiorAB = (a+b+abs(a-b))/2
greatest_number = (int(maiorAB)+c+abs(maiorAB-c))/2
print("{} eh o maior".format(int(greatest_number)))